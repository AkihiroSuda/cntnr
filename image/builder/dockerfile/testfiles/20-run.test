FROM alpine:3.7
RUN ["/install.sh", "-a"]
RUN /install.sh -a