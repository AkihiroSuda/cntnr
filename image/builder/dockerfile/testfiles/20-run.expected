FROM alpine:3.7
RUN "/install.sh" "-a"
RUN "/bin/sh" "-c" "/install.sh -a"